test/test_quantized.py

```diff
@@ -402,14 +456,14 @@ def test_qlinear_unpack(self, W):
 )
 class TestQuantizedConv(unittest.TestCase):
     """Tests the correctness of quantized convolution op."""
-    @given(batch_size=st.integers(1, 3),
+    @given(batch_size=st.integers(1, 2),
```

Comment:
What is the reason for reducing the amount of randomization? Is it the testing time?

Reply:
Yes. I was getting errors like the following:
> hypothesis.errors.Timeout: Ran out of time before finding a satisfying example for test_qconv. Only found 2 examples in 71.95s.

Reply:
Your tests perform pretty heavy computation in python -- they will take a very long time.

Reply:
I agree the test itself is heavy.